portal/templates/profile/profile_macros.html

```diff
@@ -970,14 +968,20 @@ <h5 class="muted" v-show="!isSubStudyTriggersResolved()">{{_("Actions Required (
                         <div id="postTxResolutionContainer" v-show="isSubStudyTriggersResolved()"></div>
                     </div>
                 </div>
+                <div class="study-error error-message" v-show="hasSubStudyStatusErrors()">
+                    <b>{{_("Errors: ")}}</b>
```

Comment:
nitpick: I try to be consistent with whitespace (and sometimes punctuation) for strings passed for translation, so we don't end up with a unique string to translate for each variant
```suggestion
                    <b>{{_("Errors:")}} </b>
```

Reply:
thank you, fixed.